src/webgl/p5.RendererGL.js

```diff
@@ -1389,27 +1122,27 @@ p5.RendererGL = class RendererGL extends Renderer {
 
   updatePixels() {
     const fbo = this._getTempFramebuffer();
-    fbo.pixels = this._pixelsState.pixels;
+    fbo.pixels = this.pixels;
     fbo.updatePixels();
-    this._pInst.push();
-    this._pInst.resetMatrix();
-    this._pInst.clear();
-    this._pInst.imageMode(constants.CENTER);
-    this._pInst.image(fbo, 0, 0);
-    this._pInst.pop();
+    this.push();
+    this.resetMatrix();
+    this.clear();
+    this.states.imageMode = constants.CORNER;
```

Comment:
Turns out the updatePixels problems were because setting the imageMode to CENTER does nothing when calling `this.image` as opposed to the user-exposed `image()`. I think that's fine, and actually probably for the best that our internal implementations depend less on state, but we should watch out for more cases like this left over from before.

Reply:
Technically we also don't need this line since it'll use corner mode regardless. Might be worth keeping for clarity, although then it's misleading because it looks like something that you can change when you actually can't. Maybe I should remove it?

Reply:
Yeah, that should be removed if it's doing nothing, I kept it around because I didn't know if it was actually needed or not and I just translated from what was before.